src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -2264,7 +2261,7 @@ bool ShenandoahHeap::commit_bitmap_slice(ShenandoahHeapRegion* r) {
   size_t len = _bitmap_bytes_per_slice;
   char* start = (char*) _bitmap_region.start() + off;
 
-  if (!os::commit_memory(start, len, false)) {
```

Comment:
I think this should probably be `mtGC`.  @shipilev, you don't happen to know whether this should be accounted under the Java heap? Thank you.

Reply:
This is bitmap chunk, so it is not Java heap, it is `mtGC`.

Reply:
Fixed.